src/screens/ContactScreen/index.js

```diff
@@ -106,17 +109,23 @@ const ContactScreen = ({navigation}) => {
   const handleCreateChannel = async () => {
     try {
       const mappingUserName = selectedUsers?.map((user) => user?.username).join(',');
+      const mappingUserId = selectedUsers?.map((user) => user?.user_id).join(',');
       let image = DEFAULT_PROFILE_PIC_PATH;
       if (selectedUsers.length === 1) {
         image = selectedUsers[0]?.profile_pic_path;
       }
       const dataSelected = {
         user: {
           name: mappingUserName,
-          image
+          image,
+          userId: mappingUserId
         }
       };
-      createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+      if (sourceScreen === ANONYMOUS) {
+        createAnonymousChat(dataSelected);
+      } else {
+        createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This new code introduces the creation of an anonymous chat if the source screen is `ANONYMOUS`. This is a good feature addition, but there seems to be a potential issue with the `createAnonymousChat` and `createSignChat` functions. These functions are called without any `.then()` or `.catch()` blocks, or `await` keyword, suggesting they might be asynchronous. If these functions are indeed asynchronous and return a Promise, you should handle them properly to catch any errors they might throw.
```diff
-        if (sourceScreen === ANONYMOUS) {
-          createAnonymousChat(dataSelected);
-        } else {
-          createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
-        }
+        if (sourceScreen === ANONYMOUS) {
+          await createAnonymousChat(dataSelected);
+        } else {
+          await createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+        }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->